templates/etc/login_defs.erb

```diff
@@ -0,0 +1,143 @@
+<%
+  bool_to_yesno = {
+    true => 'yes',
+    false => 'no'
+  }
+
+  if !@env_hz.nil? then
+    t_env_hz = @env_hz.dup
+    if t_env_hz !~ /^HZ=/ then
+      t_env_hz = "HZ=#{t_env_hz}"
+    end
+  end
+
+  if !@env_tz.nil? then
+    t_env_tz = @env_tz.dup
+    if t_env_tz !~ /^(\/|TZ=)/ then
+      t_env_tz = "TZ=#{t_env_tz}"
+    end
+  end
+-%>
+CHFN_AUTH <%= bool_to_yesno[@chfn_auth] %>
+CHFN_RESTRICT <%= @chfn_restrict %>
+CHSH_AUTH <%= bool_to_yesno[@chsh_auth] %>
+<% if !@console.nil? then -%>
+CONSOLE <%= @console.join(':') %>
+<% end -%>
+<% if !@console_groups.nil? then -%>
+CONSOLE_GROUPS <%= @console_groups.join(',') %>
```

Comment:
(1) Does not match String type
(2) If it is supposed to be an Array[String] need to check if it is empty, in addition to the nil check.